### ISSUE TITLE
[BUG] Offline HEMCO simulation doesn't work

### ISSUE THREAD

**Issue by cdholmes** [geoschem/geos-chem-unittest#15]
The run directory created by the unittester for standalone HEMCO doesn't work out of the box. 
After creating the run directory, the only change that I have made to the unittest run directory is to change the root data directory for local paths.

After successfully reading several files, the simulation fails while looking for an emission file for year 1714, which doesn't exist. I don't know why it's looking for that year, since it isn't specified anywhere in the HEMCO configuration files or run directory.  

I've encountered the problem in version 12.6.1. None of the subsequent commit messages in the git log appear to address this. 

The relevant parts of the HEMCO.log file are pasted below, with some initial lines for context. The first error is marked '***'.

```
Use emissions extension `Volcano`:
 - use the following species (Name, HcoID, Scaling relative to kgS):
 SO2,           38 ,    2.000000
 Apply scale field: none
  - Emissions data source is AeroCom
  - Emit eruptive emissions as category           51
  - Emit degassing emissions as category           52
Calculate emissions at 2015-04-01 00:00:00
HEMCO WARNING: Data is treated as unitless, but file attribute suggests it is not: m+2 s-2. File: /data/ExtData/GEOS_4x5/MERRA2/2015/01/MERRA2.20150101.CN.4x5.nc4
--> LOCATION: HCOIO_READ_STD (hcoio_read_std_mod.F90)
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
***HEMCO ERROR: Cannot find file for current simulation time: /data/ExtData/HEMCO/OFFLINE_BIOVOC/v2019-08/0.5x0.625/1714/04/biovoc_05x0625.17140401.nc - Cannot get field BIOGENIC_ACET. Please check file name and time (incl. time range flag) in the config. fi
ERROR LOCATION: HCOIO_READ_STD (hcoio_read_std_mod.F90)
ERROR LOCATION: HCOIO_DataRead (hcoio_dataread_mod.F90)
ERROR LOCATION: ReadList_Fill (hco_readlist_mod.F90)
ERROR LOCATION: ReadList_Read (hco_readlist_mod.F90)
ERROR LOCATION: HCO_RUN (hco_driver_mod.F90)

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
HEMCO ERROR: Error encountered in routine HCOIO_Read_Std!
ERROR LOCATION: HCOIO_READ_STD (hcoio_read_std_mod.F90)
ERROR LOCATION: HCOIO_DataRead (hcoio_dataread_mod.F90)
ERROR LOCATION: ReadList_Fill (hco_readlist_mod.F90)
ERROR LOCATION: ReadList_Read (hco_readlist_mod.F90)
ERROR LOCATION: HCO_RUN (hco_driver_mod.F90)

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
HEMCO ERROR: Error encountered in routine "Hco_Run", phase 1!
ERROR LOCATION: HCOI_SA_Run (in module HEMCO/Standalone/hcoi_standalone_mod.F90)

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
HEMCO ERROR: Error encountered in routine "HCO_Sa_Run"!
ERROR LOCATION: HCOI_StandAlone_Run (in module HEMCO/Interfaces/hcoi_standalone_mod.F90

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by yantosca** [comment]
Hi Chris.  Thanks for writing.  I was able to reproduce your issue in 12.6.1 and 12.6.2.  Stay tuned, I'll try to look for a fix.

In 12.7.0, we are going to be adding some updates to how HEMCO handles searching for time indexes in files.  This should hopefully make HEMCO more robust and take less time for file I/O.

**Comment by msulprizio** [comment]
Please see our separate HEMCO repository (https://github.com/geoschem/HEMCO), which will be included as a Git submodule in GEOS-Chem 13.0.0. Any issues that were in the previous HEMCO standalone runs have been resolved there. The HEMCO repository is current with GEOS-Chem 12.8.0. HEMCO standalone run directories may be created with the `run/createRunDir.sh` script.
